app/models/validations/household_validations.rb

```diff
@@ -188,7 +188,7 @@ def validate_person_age_and_gender_match_economic_status(record, person_num)
   end
 
   def validate_partner_count(record)
-    partner_count = (2..8).count { |n| record.public_send("relat#{n}")&.zero? }
+    partner_count = (2..8).count { |n| tenant_is_partner?(record.public_send("relat#{n}")) }
```

Comment:
`record["relat#{n}"]` would that not work?

Reply:
Yep it would. Prefer that?

Reply:
Dunno, if it works the same with nil values then maybe?